fuel/converters/toronto_face_database.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+import h5py
+import numpy
+
+from scipy.io import loadmat
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def toronto_face_database(input_directory, save_path):
+"""Converts the Toronto Face Database to HDF5.
+
+    ToDo
+
+    Parameters
+    ----------
+    input_directory : str
+        Directory in which the required input files reside.
+    save_path : str
+        Where to save the converted dataset.
+
+    """
+    h5file = h5py.File(save_path, mode="w")
+
+    tfd = loadmat(os.path.join(input_directory, 'TFD_48x48.mat'))
+    
+    folds = tfd['folds']
+    features = tfd['images'].reshape([-1, 1, 48, 48])
+    labs_ex = tfd['labs_ex']
+    labs_id = tfd['labs_id']
```

Comment:
I'd be more explicit with the choice of source name. Something like `'expression_targets'` and `'identity_targets'` would work for me.